lib/output/utils.js

```diff
@@ -74,6 +74,52 @@ const namedSetNew = Symbol("named property set new");
 const namedSetExisting = Symbol("named property set existing");
 const namedDelete = Symbol("named property delete");
 
+/**
```

Comment:
I am not interested in this addition. Please remove it.

Reply:
Which? The&nbsp;JSDoc or&nbsp;the&nbsp;entire `utils.importStar(…)`&nbsp;function?

Reply:
The importStar function itself.